travis-results-4.txt

```diff
@@ -0,0 +1,205 @@
+>>> Finished with service_mesh book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on metering book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with metering book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on web_console book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with web_console book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on monitoring book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with monitoring book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on authentication book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with authentication book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on logging book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 178: callout list item index: expected 7, got 6
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 179: callout list item index: expected 8, got 7
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 180: callout list item index: expected 9, got 8
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 181: callout list item index: expected 10, got 9
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 182: callout list item index: expected 11, got 10
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 183: callout list item index: expected 12, got 11
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 183: no callout found for <12>
+>>> Finished with logging book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on scalability_and_performance book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with scalability_and_performance book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on cli_reference book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with cli_reference book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on backup_and_restore book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+asciidoctor: WARNING: includes/test-module-2.adoc: line 6: id assigned to section already in use: test-module-1_backup-etcd
+[31m[1mID "test-module-1_backup-etcd" is duplicated in the source content[0m
+[31m[1mbuild/master.xml fails to validate[0m
+>>> Finished with backup_and_restore book in openshift-enterprise <<<
+---------------------------------------
```

Comment:
This warning is for the incorrect ID in test-module-2, which uses the same ID as in test-module-1.